lxl-web/src/routes/(app)/[[lang=lang]]/(find)/FacetGroup.svelte

```diff
@@ -74,13 +74,15 @@
 							{#if 'selected' in facet}
 								<!-- howto A11y?! -->
 								<span class="sr-only"
-									>{facet.selected ? $page.data.t('search.selectedFilter') : ''}</span
+									>{facet.selected ? $page.data.t('search.activeFilters') : ''}</span
```

Comment:
This is the accessible label for a single checked checkbox in the (not yet implemented) multiple (OR) filters. I think "valt filter" is a more accurate translation than "aktiva filter". No biggie as long as it's non-existent however...